### ISSUE TITLE
Adding chromosome-wide prediction

### ISSUE THREAD

**Issue by tanjimin** [tanjimin/C.Origami#24]
A lot of request was received on chromosome-wide predictions. I will add this feature soon.

**Comment by GMFranceschini** [comment]
Dear Jimin, I'd like to express also my interest for this feature.
In the meantime, do you think there is any workaround I might experiment myself to get a chromosome wide prediction? (Of course without any guarantee of success, but any suffestion is welcome)

**Comment by rebeccaronai** [comment]
I would also like to express my interest for this feature. It will help immensely in my research!

**Comment by tanjimin** [comment]
Hi @GMFranceschini, and @rebeccaronai  the general idea is to predict multiple regions and merge them along the genome axis. The overlapping region should be averaged. The result will look like a strip on the diagonal with little to no artifacts. Sorry I haven't got to it yet and will probably do it in the near future.

If you implemented it, feel free to submit a pull request. I would be happy to check for you and merge the changes.

**Comment by GMFranceschini** [comment]
Thank you for your answer @tanjimin, I think I got the idea. 

However, do you think it is possible to predict off-diagonal contacts as well? I mean with the aim of getting a full square contact matric for each chromosome, and not only the neighborhood of the diagonal. I don't know if the model contemplates such scenarios (large distance, sparse contacts...)

**Comment by tanjimin** [comment]
Yes, interactions beyond 2mb are large scale and usually represent compartments. You can probably predict them with some classic polymer-based models. C.Origami currently don't have the capacity at that level.

**Comment by GMFranceschini** [comment]
Thanks a lot for your insights. I was indeed interested in compartment calling as the ultimate goal, but I see now that it is beyond the scope of your model.
